src/ServiceControl/Monitoring/MonitoringDataPersister.cs

```diff
@@ -101,7 +128,9 @@ public async Task WarmupMonitoringFromPersistence()
                     while (await endpointsEnumerator.MoveNextAsync().ConfigureAwait(false))
                     {
                         var endpoint = endpointsEnumerator.Current.Document;
-                        monitoring.DetectEndpointFromPersistentStore(endpoint.EndpointDetails, endpoint.Monitored);
+                        var heartbeatsEnabled = !endpoint.HeartbeatsEnabled.HasValue || endpoint.HeartbeatsEnabled.Value;
```

Comment:
Is `endpoint.HeartbeatsEnabled ?? true` easier to read?

Reply:
Agreed